certbot-apache/certbot_apache/configurator.py

```diff
@@ -2516,4 +2522,16 @@ def deploy_autohsts(self, lineage):
         self._autohsts_save_state()
 
 
+def find_ssl_apache_conf(prefix):
+    """
+    Find a TLS Apache config in the dedicated storage.
+    :param str prefix: prefix of the TLS Apache config to find
```

Comment:
[super nit]: adding "file" after "...Apache config" would make reader to immediately grok what's going on in here.